index.html

```diff
@@ -4,15 +4,14 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <script type="module" src="/src/JS/main.js"></script>
-    <link rel="stylesheet" href="/static/css/main.css" />
+    <link rel="stylesheet" href="static/css/main.css" />
+    <script type="module" src="src/JS/main.js"></script>
```

Comment:
Thanks for fixing " /  " problem on pages

Reply:
No problem :)